app/models/proposal.rb

```diff
@@ -167,5 +167,11 @@ def restart
     Dispatcher.deliver_new_proposal_emails(self)
   end
 
+  def changed_fields comment_text
+    Comment.new({comment_text: comment_text, 
+                 update_comment: true, 
+                 user_id: self.requester_id, 
+                 proposal_id: self.id}).save
```

Comment:
Also, I would use `self.comments.create(...)` so that it takes care of the association.